packages/hash/api/src/model/link.model.ts

```diff
@@ -241,6 +242,33 @@ class __Link {
     return await this.fetchDestination(client);
   }
 
+  async update(
+    client: DBClient,
+    params: { updatedIndex: number; updatedByAccountId: string },
+  ) {
+    const { updatedIndex, updatedByAccountId } = params;
+
+    /** @todo: implement way of updating a link's index without deleting the current one and creating a new one */
```

Comment:
Should be done as part of [this follow up task](https://app.asana.com/0/1201095311341924/1201817757879408/f)